src/cbmc/Makefile

```diff
@@ -5,7 +5,6 @@ SRC = all_properties.cpp \
       cbmc_languages.cpp \
       cbmc_main.cpp \
       cbmc_parse_options.cpp \
-      fault_localization.cpp \
```

Comment:
Can those Makefile changes please be in the same commit as the actual removal of the file?